optimum/exporters/tflite/model_configs.py

```diff
@@ -97,3 +97,11 @@ def inputs(self) -> List[str]:
 
 class DebertaV2TFLiteConfig(DebertaTFLiteConfig):
     pass
+
+
+class ResNetTFLiteConfig(VisionTFLiteConfig):
+    NORMALIZED_CONFIG_CLASS = NormalizedConfigManager.get_normalized_config_class("resnet")
+
+    @property
+    def inputs(self) -> Dict[str, Dict[int, str]]:
+        return {"pixel_values": {0: "batch_size", 1: "num_channels", 2: "height", 3: "width"}}
```

Comment:
Inputs shjould return a list of the input names:
```suggestion
        return ["pixel_values"]
```
Do you still have the error when changing to that?

Reply:
Same issue persists.

Reply:
Maybe my `tflite_config` is initialized in the wrong way in my test file? I could not infer it from the existing TFLite tests either.